site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
+
+Chrome 91 adds the following features to the extensions platform.
+
+#### ES modules for service workers
+
+JavaScript supports modules in service workers. Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts. (https://www.chromestatus.com/guide/edit/4609574738853888)
+
+#### chrome.action.getUserSettings() available
+
+The new chrome.action.getUserSettings() API allows extensions to determine if the user has pinned the extension to the main toolbar.
+
+### Chrome 90 (stable on April 13, 2021)
+
+#### New chrome.scripting.removeCSS() method (MV3 only)
+
+The `chrome.scripting.removeCSS()` method lets extensions remove CSS that was previously inserted via
+chrome.scripting.insertCSS(). It replaces the MV2 method `chrome.tabs.removeCSS()`.
+
+#### chrome.scripting.executeScript() results now include frameId
+
+Results returned from `chrome.scripting.executeScript()` now include the frameId.  The frameId indicates the frame from which the result is from, allowing extensions to easily associate results with the individual frames when injecting in multiple frames.
```

Comment:
Should we put frameId in backticks, too?  And _possibly_ (but not necessarily) link to https://developer.chrome.com/docs/extensions/reference/webNavigation/#a-note-about-frame-ids?

Side note (not for this PR): We should pull frame IDs and tab IDs into a central article; both are used far more broadly than just the tabs / webNavigation APIs...

Reply:
```suggestion
Results returned from [`chrome.scripting.executeScript()`][execute-script] now include the [`frameId`][frame-id].  The frameId indicates the frame from which the result is from, allowing extensions to easily associate results with the individual frames when injecting in multiple frames.

[execute-script]: https://developer.chrome.com/docs/extensions/reference/scripting/#method-executeScript
[frame-id]:  https://developer.chrome.com/docs/extensions/reference/webNavigation/#a-note-about-frame-ids
```